spyderlib/widgets/dicteditor.py

```diff
@@ -1075,7 +1076,23 @@ def copy(self):
         for idx in self.selectedIndexes():
             if not idx.isValid():
                 continue
-            clipl.append(to_text_string(self.delegate.get_value(idx)))
+            obj = self.delegate.get_value(idx)
+            # check if it's a numpy array, and if so make sure to copy the whole
+            # thing in a tab separated format
+            try:
+                import numpy as np
+            except ImportError:
+                pass # skip this part

```

Comment:
You need it to check if the object is a ndarray. You you need it in order to know if you need it !